content/hi/zero-trust-architecture.md

```diff
@@ -0,0 +1,34 @@
+---
+title: झिरो ट्रस्ट आर्किटेक्चर (Zero Trust Architecture)
+status: Completed
+category: संकल्पना
+tags: ["security", "", ""]
+---
+
+## यह क्या है
+
+झिरो ट्रस्ट आर्किटेक्चर आईटी प्रणाली के डिजाइन और कार्यान्वयन के लिए एक दृष्टिकोण निर्धारित करता है
+जहां विश्वास पूरी तरह से हटा दिया गया है। 
+मूल सिद्धांत "कभी भरोसा न करें, हमेशा सत्यापित करें", उपकरण या प्रणाली स्वयं,
+प्रणाली के अन्य घटकों से संचार करते समय, ऐसा करने से पहले हमेशा स्वयं को सत्यापित करें। 
+आज कई नेटवर्क में, निगमित नेटवर्क के भीतर, प्रणाली और उपकरण के अंदर स्वतंत्र रूप से एक दूसरे के साथ संवाद कर सकते हैं
+क्योंकि वे निगमित नेटवर्क परिधि की विश्वसनीय सीमा के भीतर हैं।
+झिरो ट्रस्ट आर्किटेक्चर विपरीत दृष्टिकोण लेता है, हालांकि नेटवर्क परिधि के अंदर,
+किसी भी संचार से पहले प्रणाली के भीतर घटकों को पहले सत्यापन पास करना होगा।
+
+## समस्या यह संबोधित करता है
+
+पारंपरिक ट्रस्ट आधारित दृष्टिकोण के साथ जहां प्रणाली और उपकरण जो निगमित नेटवर्क परिधि के भीतर मौजूद हैं, धारणा यह है कि क्योंकि विश्वास है, कोई समस्या नहीं है। 
```

Comment:
for almost all sentences in this PR, please align them correctly without spaces and extra new lines so that it looks like paragraphs.

Reply:
sure @Garima-Negi